lib/app/result.js

```diff
@@ -1,15 +1,7 @@
 'use strict';
 
-var SimpleStore = require('fh-wfm-simple-store');
+var store = require('../store-init.js');
 
 module.exports = function(mediator) {
-  var resultsStore = new SimpleStore('result');
-
-  resultsStore.list().then(function(res) {
-    if (!res || res.length === 0) {
-      resultsStore.init();
-    }
-
-    resultsStore.listen(':cloud', mediator);
-  });
+  store.init('result', null, mediator);
```

Comment:
`null` here might break the in-memory implementation :sweat_smile:.

We can go ahead and test with it but `[]` is probably more meaningful.